examples/kube-mcp-server/tasks/scale-down-deployment/setup.sh

```
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+# Create namespace and a deployment with initial replicas
+kubectl delete namespace scale-down-test --ignore-not-found
+kubectl create namespace scale-down-test
+kubectl create deployment web-service --image=nginx --replicas=2 -n scale-down-test
+# Wait for initial deployment to be ready
+for i in {1..30}; do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)